resources/views/js-modal.blade.php

```diff
@@ -48,9 +48,9 @@ class="modal-close"
                 @endif
 
                 @if ($title)
-                    <h1 class="{{ $titleClass }}">
+                    <h2 class="{{ $titleClass }}">
```

Comment:
does this affect styling? as h1 will be styled differently from h2. Check with designs whether it should look like a h1 tag and if so add the `header-1` class to it

Reply:
designs are h2 now and projects have been adjusted to it already, so the change is fine in this case